Tests/Unit/Microsoft365DSC/Microsoft365DSC.IntuneAccountProtectionLocalAdministratorPasswordSolutionPolicy.ps1

```diff
@@ -0,0 +1,343 @@
+[CmdletBinding()]
```

Comment:
This file should be named Microsoft365DSC.IntuneAccountProtectionLocalAdministratorPasswordSolutionPolicy.Tests.ps1. Currently the .Tests is missing at the end of the file.